src/css/components/tooltip.css

```diff
@@ -0,0 +1,9 @@
+.tooltip {
```

Comment:
Nice simplification of styles :+1: 

Please add a `villain` prefix to the `className` so it doesn't cause conflicts with external / global styles

Reply:
okay i'll add it